components/is-analytics-solutions/org.wso2.carbon.sp.is.analytics.solutions/widgets/AverageSessionDuration/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "AverageSessionDuration",
```

Comment:
Name should be as per the standard "Is Analytics Average Session Duration", while id is "IsAnalyticsAverageSessionDuration". Do the change in directory name as well